vircadia/vircadia
# Update webapp branch to recent master

- Merge latest master into webapp branch.
- Rename instances of HifiSockAddr to SockAddr in webapp branch.
- Fix an erroneous WebRTC domain server log warning message.

##### QA
Check that the tests in https://github.com/vircadia/vircadia-web-sdk/pull/20 run OK.
